src/axes/categoryAxis.ts

```diff
@@ -112,18 +117,22 @@ namespace Plottable.Axes {
     }
 
     protected _getTickValues() {
-      return this.getDownsampleInfo()[0];
+      return this.getDownsampleInfo().domain;
     }
 
     /**
      * Take the scale and drop ticks at regular intervals such that the resultant ticks are all a reasonable minimum
      * distance apart. Return the resultant ticks to render, as well as the new stepWidth between them.
      *
-     * @return [downsampled domain, new stepWidth]
+     * @param scale the scale being downsampled. Defaults to this Axis' scale.
```

Comment:
Sorry to nitpick on this, but remember this is a public method so let's keep comments more formal
`@param {Scales.Category} scale - The scale being downsampled. Defaults to this Axis' scale.`